python/strings_udf/strings_udf/_typing.py

```diff
@@ -229,6 +229,7 @@ def resolve_count(self, mod):
     "isnumeric",
     "istitle",
 ]
+string_binary_funcs = ["strip", "lstrip", "rstrip"]
```

Comment:
This name is somewhat misleading. There are other string binary functions that we already have implemented, including operators and things like `find` and `contains`. Is this list meant to contain binary string functions that also return a string, or is it even more specific than that?

Reply:
Renaming this to `string_return_attrs`